optimum/exporters/neuron/convert.py

```diff
@@ -147,9 +147,9 @@ def export(
     **kwargs,
```

Comment:
I am not a big fan of kwargs when it can be avoided. Are you doing this because it should be able to work with both neuron and neuronx?

Reply:
Yes exactly, they have different options for auto-casting, that's why I used kwargs to share to the export func.

Reply:
Do we still have **kwargs?

Reply:
I hesitated to put `disable_fast_relayout` as it is only for INF1. But it's better than kwargs. I will put `disable_fast_relayout`  and remove kwargs then.